patches/tModLoader/Terraria/Chest.cs.patch

```diff
@@ -146,9 +146,19 @@
  		}
  
  		public void SetupShop(int type) {
-@@ -2611,6 +_,7 @@
+@@ -2610,7 +_,17 @@
+ 				if (Main.player[Main.myPlayer].ZoneGlowshroom && num < 39)
  					array[num++].SetDefaults(4921);
  			}
++			foreach (ModPylon pylon in ModContent.GetContent<ModPylon>()) {
```

Comment:
pretty sure this doesn't actually work. `ModContent` only works on the actual type of an instance, not on its parent types. 

Should just have `PylonLoader.ModPylons`